<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generate playlist</string>
  </property>
  <widget class="QCheckBox" name="chk_live">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Live</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_vod">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VOD</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_series">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Series</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_categories">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Categories to include:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_url">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_url">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>21</y>
     <width>391</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>c:\some\path\playlist.m3u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_browse">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>19</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_group_filter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Group filter: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_group_filter">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Filter the groups to export, separated by commas.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_title_filter">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Filter the titles to export, separated by commas.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_title_filter">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Title filter:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
